server/context.go

```diff
@@ -147,7 +147,9 @@ func getDefaultCacheSize() uint64 {
 	mem := sigar.Mem{}
 	if err := mem.Get(); err != nil {
 		if log.V(1) {
-			log.Infof("can't retrieve system memory information (%s), setting default rocksdb cache size to %dMB", err, defaultCacheSize>>20)
+			log.Infof("can't retrieve system memory information (%s)\n"+
+				"\tsetting default rocksdb cache size to %dMB",
```

Comment:
Yep. We split the log file at log message header boundaries and grab everything from the start of a message header to the beginning of the next as a single message.